### PR TITLE
DOC: rework ethercat terminal docs page + fix width

### LINKED ISSUE #3: Show or link to example generated documentation
> Maybe an example docs page might be good to include with this project, so people roughly know what it's attempting to do? ...

_Originally posted by @tangkong in https://github.com/pcdshub/twincat-docs-template/pull/2#pullrequestreview-1241170395_

### LINKED ISSUE #7: Update EtherCAT to use the same tree as plc-summary
This
https://pcdshub.github.io/lcls-plc-qrix-motion/master/lcls-plc-qrix-motion_ethercat.html
Should look like this:
https://pcdshub.github.io/plc-summary/ethercat.html#pcdshub-lcls-plc-qrix-motion

But also note:
https://github.com/pcdshub/plc-summary/issues/7

### PR DESCRIPTION
**Pull request by klauer**
Closes #3 
Closes #7 

Tested locally.

Looks like this:
image
(scrolling down to the syncunit)
image

### CODE REVIEW

**Review by tangkong — APPROVED** [review]
one of those "results speak for themselves" approvals.  Really cool tree views 👍

File: source/_static/expand-collapse.svg
Comment (ZLLentz) [inline source/_static/expand-collapse.svg:1]: Is there a story behind this pure white svg file?

**Review by ZLLentz — APPROVED** [review]

File: source/_static/expand-collapse.svg
Comment (tangkong) [reply source/_static/expand-collapse.svg:1]: If you highlight it, it's actually a small transparent thing and github fills it with white (I think)

Comment (klauer) [reply source/_static/expand-collapse.svg:1]: Yeah, it's this 
image
and magically _also_ this 
image

Comment (ZLLentz) [reply source/_static/expand-collapse.svg:1]: Story understood: do not use github as an image viewer

**Pull request merged**